### ISSUE TITLE
Passed a file-system folder or subtree to custom indexer (rather than individual files)

### ISSUE THREAD

**Issue by randym32** [CoatiSoftware/Sourcetrail#1209]
Summary:
I would like an mode for the custom indexer to be passed a file-system folder or subtree, rather than individual files.

I have created my own source indexers.  It has a mode that scans all of the files in a subtree.  When it scans 400 files, it takes only a second; when it scans 1400 files, it takes less than 6.    

When I configure Sourcetrail to use the indexer, the process takes 2-3 minutes to scan the 400 files, and 8 minutes to scan scan the 1400 files.  (It also takes several minutes after Sourcetrail says the process has completed before it is ready to show anything.)

**Comment by mlangkabel** [comment]
No, this is currently not possible. You could create a "custom" source group that just indexes a single `.txt` file that lists all the folders that you want your indexer to scan.
